conf/config.yaml.sample

```diff
@@ -64,11 +64,41 @@ cortx:
     transport_type: libfab
     ios:
       endpoints:
-      - motr://data1-node1.cortx-cluster.com:3232
+      - tcp://data-node1:28002
+      - tcp://data-node1:28003
+      - tcp://data-node2:28002
+      - tcp://data-node2:28003
+      - tcp://data-node3:28002
+      - tcp://data-node3:28003
+    confd:
+      endpoints:
+      - tcp:data-node1:28004
```

Comment:
Need to update "tcp://" format at all the places .

Reply:
Done.